diff -r 3d87a62c5939 src/template_dictionary.cc
--- a/src/template_dictionary.cc	Wed May 09 11:41:21 2012 +0100
+++ b/src/template_dictionary.cc	Wed May 09 12:50:11 2012 +0100
@@ -44,7 +44,7 @@
 #include <string>
 #include <utility>              // for pair<>
 #include <vector>
-
+#define	va_copy(to, from)	((void)((to) = (from)))
 #include "base/arena-inl.h"
 #include "base/thread_annotations.h"
 #include "indented_writer.h"
