--- a/single_include/catch2/catch.hpp	2022-04-12 21:38:17
+++ b/single_include/catch2/catch.hpp	2024-05-03 12:54:23
@@ -69,6 +69,11 @@
 // See e.g.:
 // https://opensource.apple.com/source/CarbonHeaders/CarbonHeaders-18.1/TargetConditionals.h.auto.html
 #ifdef __APPLE__
+#  ifndef __clang__
+#    if !__has_builtin(__has_extension)
+#      define __has_extension(x) 0
+#    endif
+#  endif
 #  include <TargetConditionals.h>
 #  if (defined(TARGET_OS_OSX) && TARGET_OS_OSX == 1) || \
       (defined(TARGET_OS_MAC) && TARGET_OS_MAC == 1)
